### PR TITLE
FS-80 - Api services, models

### PR DESCRIPTION
**Pull request by grands14320**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
     "coverage": "vitest run --coverage"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/icons-material": "^5.11.0",
```
Comment (kaspermroz) [inline package.json:19]: Please drop this dependencies and use SVG icons exported from figma (place them inside `src/assets/icons`

File: src/components/shared/Logo/Logo.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export const Logo = () => {
+  return (
+    <svg
```
Comment (kaspermroz) [inline src/components/shared/Logo/Logo.tsx:3]: move it to separate SVG file

File: src/pages/LandingPage/LandingPage.styles.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
+.play-btn {
+  box-sizing: border-box;
+  height: 50vh;
+  position: absolute;
+
+  border-color: transparent transparent transparent #ffd502;
+  /*border-color: transparent transparent transparent red;*/
```
Comment (kaspermroz) [inline src/pages/LandingPage/LandingPage.styles.css:7]: use theme color

File: src/models/enums/LocalStorage.ts [hunk 4]
```diff
@@ -1,3 +1,3 @@
-export enum Token {
+export enum LocalStorage {
   TOKEN = "CINEMON-TOKEN",
```
Comment (kaspermroz) [inline src/models/enums/LocalStorage.ts:2]: pls avoid using TS enums, consts are just fine (or unions)

File: src/main.tsx [hunk 5]
```diff
@@ -1,18 +1,25 @@
...
+    element: <LandingPage />,
   },
 ]);
 
+const queryClient = new QueryClient();
+
 ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
   <React.StrictMode>
-    <RouterProvider router={router} />
+    <QueryClientProvider client={queryClient}>
```
Comment (kaspermroz) [inline src/main.tsx:20]: I think I have this injected already in App?

File: src/components/shared/navbar/Navbar.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+import LanguageIcon from "@mui/icons-material/Language";
+import PersonIcon from "@mui/icons-material/Person";
+import { FullLogo } from "../FullLogo";
+
+export const Navbar: FC = () => {
+  return (
+    <div className="p-4 flex flex-row justify-between shadow-lg">
+      <FullLogo />
+      <div className="flex flex-row items-center gap-x-12">
+        {/*https://tailwind-elements.com/docs/standard/forms/search/*/}
```
Comment (kaspermroz) [inline src/components/shared/navbar/Navbar.tsx:12]: drop comment

File: src/main.tsx (hunk 5 above)
Comment (grands14320) [reply src/main.tsx:20]: smth was fcked and i moved it to the top lvl

File: .gitignore [hunk 7]
```diff
@@ -26,4 +26,4 @@ dist-ssr
 # Test
 coverage
 
-.env
+.env.example
```
Comment (kaspermroz) [inline .gitignore:29]: don't do that, this file is ignored on purpose :)
.env - local machine + deployment stages
.env-example - example on the repo to copy and create your own .env file

File: tailwind.config.cjs [hunk 8]
```diff
@@ -2,7 +2,12 @@
 module.exports = {
   content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        "company-yellow": "#FFD502",
+        variable: "var(--company-yellow)"
```
Comment (kaspermroz) [inline tailwind.config.cjs:8]: I feel like this is redundant, you can just do tailwind magic to use this color?

File: src/components/shared/Navbar/Navbar.tsx [hunk 9]
```diff
@@ -0,0 +1,35 @@
+import { FC } from "react";
+import { ReactComponent as SearchIcon } from "~/assets/icons/SearchIcon.svg";
```
Comment (kaspermroz) [inline src/components/shared/Navbar/Navbar.tsx:2]: nice ✨

File: src/assets/icons/CinemonLogo.svg [hunk 10]
```diff
@@ -0,0 +1,3 @@
+<svg width="93" height="58" viewBox="0 0 93 58" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (kaspermroz) [inline src/assets/icons/CinemonLogo.svg:1]: Isn't this one blank? GitHub shows nothing there

File: tailwind.config.cjs (hunk 8 above)
Comment (kaspermroz) [reply tailwind.config.cjs:8]: Like can you add more vars to this?

File: .gitignore (hunk 7 above)
Comment (grands14320) [reply .gitignore:29]: no idea how did it get there

File: src/assets/icons/CinemonLogo.svg (hunk 10 above)
Comment (grands14320) [reply src/assets/icons/CinemonLogo.svg:1]: no fill colour assigned

File: src/AxiosApiV1.tsx [hunk 11]
```diff
@@ -19,7 +19,7 @@ export const isTokenExpired = (): boolean => {
 };
 
 const api = axios.create({
-  baseURL: import.meta.env.VITE_API_BASE_URL,
+  baseURL: "http://localhost:8080/api/",
```
Comment (kaspermroz) [inline src/AxiosApiV1.tsx:22]: can't change that, it will break the service. it's the env var

**Review by kaspermroz — APPROVED** [review]

**Pull request merged**